### PR TITLE
Added SuiteCRM

### PR DESCRIPTION
**Pull request by Sidd2024**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

### CODE REVIEW

File: docs/list.md [hunk 1]
```diff
@@ -292,29 +280,6 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
-        - On-Platform blog: 5
-        - Off-Platform blog: 5
-        - Docs: 4
-        - Forum answer: 3 (+2 if marked as solution)
-- **How to sign up**:
-    - You can simply visit one of their repositories [here](https://github.com/search?q=topic%3Ahacktoberfest+org%3Aopensearch-project+fork%3Atrue&type=repositories){:target="\_blank"} and pick up any available issue to submit a PR.
-    - The OpenSearch team will review the pull request and let you know if any changes are needed.
-- **Issues**: See the list of available Hacktoberfest issues [here](https://github.com/search?q=topic%3Ahacktoberfest+org%3Aopensearch-project+fork%3Atrue+label%3A%22hacktoberfest%22+is%3Aissue+is%3Aopen&type=Issues){:target="\_blank"}.
-- **Notes**: More details about the prizes [are here](https://opensearch.org/blog/community/2022/10/hacktoberfest-2022/){:target="\_blank"}. Join their [Forum](https://forum.opensearch.org/t/hacktoberfest-2022/11129){:target="\_blank"} if you have any further queries. They are also active on [Twitter](https://twitter.com/OpenSearchProj){:target="\_blank"}.
-
```
Comment (crweiner) [inline docs/list.md:317]: Can you please rebase or delete and re-fork, please? This should not be deleted.

File: docs/list.md [hunk 2]
```diff
@@ -377,6 +342,15 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
     - The Steampipe team will review the pull request and let you know if any changes are needed.
 - **Notes**: For more details [their Hacktoberfest page](https://steampipe.io/blog/hacktoberfest-2022){:target="\_blank"}.
 
+#### **SuiteCRM**
+
+- **Swag**: T-shirt, stickers, SuiteCRM-branded mug
+- **Requirements**: Raise PRs, Plenty of them!
```
Comment (crweiner) [inline docs/list.md:348]: Let's instead say something like:

```suggestion
- **Requirements**: Unsure as to number of PRs required at this time.
```

**Review by crweiner — CHANGES_REQUESTED** [review]
Hello @Sidd2024! 👋 Thank you for your PR. There are several things for you to change, please. Give me a ping when you're ready for me to take another look. Thanks! 👍

File: docs/list.md [hunk 3]
```diff
@@ -267,18 +267,6 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
...
-- **Swag**: NTC & Nautobot swag including shirt(s), stickers, mugs, and more.
-- **Requirements**: There will be different prizes for each of the following groups.    
-    - New Contributors and all qualifying video submissions.
-    - Most Diverse Contributions (qty of repos).
-    - Most lines of code changed (across repos).
-    - Most bugs closed (PRs opened and approved).
-    - Most docs updated (PRs opened and approved).
-- **How to sign up**: To participate in Hacktoberfest there's no need to sign up, all you need to do is submit pull request(s) for a qualifying repo.
-- **Notes**: Check out [here](https://blog.networktocode.com/post/hackathon/){:target="\_blank"} for more info.
-
```
[the remark this answers is not in the document]
Comment (Sidd2024) [reply docs/list.md:281]: @crweiner sure!

File: docs/list.md [hunk 4]
```diff
@@ -377,6 +342,15 @@ Appwrite is a self-hosted backend-as-a-service platform that provides developers
     - The Steampipe team will review the pull request and let you know if any changes are needed.
 - **Notes**: For more details [their Hacktoberfest page](https://steampipe.io/blog/hacktoberfest-2022){:target="\_blank"}.
 
+#### **SuiteCRM**
+
+- **Swag**: T-shirt, stickers, SuiteCRM-branded mug
+- **Requirements**: Raise PRs, Plenty of them!
+- **How to sign up**:
+    - Sign up [here](https://hacktoberfest.com/auth/){:target="\_blank"}
```
[the remark this answers is not in the document]
Comment (Sidd2024) [reply docs/list.md:350]: not mentioned on post or their github

**Pull request closed without merging**